scripts/align_z_focus.py

```diff
@@ -159,7 +159,7 @@ def main(input_dir,
         im_name_dst = get_sms_im_name(
             time_idx=meta_row['time_idx'],
             channel_name=meta_row['channel_name'],
-            slice_idx=meta_row['slice_idx_new'],
+            slice_idx=meta_row['slice_idx_new'].astype('int64'),
```

Comment:
This is fine by me. My only question is: would it make sense to do the data type conversion inside of get_sms_im_name so that you get the right data type whenever you call the function?

Reply:
Hello Jenny. That is a much better approach of resolving the issue. I have tested the modification on my data as well. Thank you for the suggestion.